server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/RecipientRewriteTableProcessor.java

```diff
@@ -223,37 +223,27 @@ static ForwardDecision removeRecipient(MailAddress recipient) {
                 .collect(ImmutableList.toImmutableList()));
         }
 
-        static ForwardDecision sendACopy(MailetContext context, MailAddress originalRecipient, List<MailAddress> forwards) {
+        static ForwardDecision sendACopy(MailetContext context,
+                                         MailAddress originalRecipient,
+                                         List<MailAddress> forwards,
+                                         LoopPrevention.RecordedRecipients recordedRecipients,
+                                         Set<MailAddress> newRecipients) {
```

Comment:
Why are forwards and original recipients different?

Reply:
for example, if the original recipient is A, forwards will be the list of mail addresses that A has to forward email to